resources/views/account/partials/address-cards.blade.php

```diff
@@ -0,0 +1,38 @@
+<x-rapidez-ct::title.lg>
+    @lang('My addresses')
+</x-rapidez-ct::title.lg>
+<div class="grid gap-5 grid-cols-1 sm:grid-cols-2 my-5">
+    <template v-if="data.customer.shipping_address?.default_billing || !data.customer.shipping_address || !data.customer.billing_address">
+        <template v-if="data.customer.shipping_address || data.customer.billing_address">
+            <x-rapidez-ct::card.address v-bind:address="data.customer.shipping_address ?? data.customer.billing_address" shipping billing check>
+                <x-rapidez-ct::button.link v-bind:href="`/account/address/${(data.customer.shipping_address ?? data.customer.billing_address).id}`">
+                    @lang('Edit')
+                </x-rapidez-ct::button.link>
+            </x-rapidez-ct::card.address>
+        </template>
+        <a href="/account/address/new" class="h-full min-h-[180px] flex flex-col items-center justify-center gap-y-2 font-medium bg-ct-border rounded max-sm:hidden">
```

Comment:
you're setting a border color but dont have a border? or does this get added by a divide-y etc?

Reply:
that's a background color :) but I guess it should be `bg-ct-inactive-200` instead.